utils/config/daos_server.yml

```diff
@@ -204,7 +203,7 @@
 #
 #  targets: 20
 #
-#  # Count of offload/helper xstreams per engine.
+#  # Number of I/O service threads (and network endpoints) per engine.
 #  # Immutable after reformat.
 #
 #  nr_xs_helpers: 20
```

Comment:
Could you please change this default to 6? We don't recommend using more than 22 cores anyways.